module/Motion/BonusBlockCarrier.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file BonusBlockCarrier.h
+ * @brief ボーナスブロックをベースエリアまで運搬する
+ * @author sugaken0528
+ */
+
+#ifndef BONUS_BLOCK_CARRIER_H
+#define BONUS_BLOCK_CARRIER_H
+
+#include "Rotation.h"
+#include "LineTracer.h"
+#include "StraightRunner.h"
+#include "Controller.h"
+
+class BonusBlockCarrier {
+ public:
+  static void carryBonusBlock();
+};
```

Comment:
staticクラスにするなら，privateにコンストラクタを記述してインスタンス化を禁止してほしい

参考：[資料](https://programming-place.net/ppp/contents/cpp/language/018.html)の`staticクラス`の項目

```suggestion
 private:
  BounusBlockCarrier(); // インスタンス化を禁止する
};
```